content/ko/docs/reference/using-api/health-checks.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 쿠버네티스 API 헬스(health) 엔드포인트
+reviewers:
+-
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+쿠버네티스 {{< glossary_tooltip term_id="kube-apiserver" text="API 서버" >}}는 현재 상태를 나타내는 API 엔드포인트를 제공한다.
+이 페이지에서는 API 엔드포인트들에 대해 설명하고 이를 사용하는 방법에 대해 설명한다.
+
+<!-- body -->
+
+## 헬스(health)를 위한 API 엔드포인트
+
+쿠버네티스 API 서버는 현재 상태를 나타내는 3가지 API 엔드포인트(`healthz`, `livez` 와 `readyz`)를 제공한다.
+`healthz` 엔드포인트는 사용 중단(deprecated) 됐으며 (Kubernetes v1.16 버전이후), 대신 보다 구체적인 `livez` 와 `readyz` 엔드포인트를 사용해야 한다.
+`livez` 엔드포인트는 `--livez-grace-period` [flag](/docs/reference/command-line-tools-reference/kube-apiserver) 옵션을 사용하여 시작 대기 시간을 지정할 수 있다.
+`/readyz` 엔드포인트는 `--shutdown-delay-duration` [flag](/docs/reference/command-line-tools-reference/kube-apiserver) 옵션을 사용하여 Gracefull shutdown 할 수 있다.
+API 서버의 `health`/`livez`/`readyz` 를 사용하는 머신은 HTTP 상태 코드에 의존해야 한다.
+상태 코드 200 은 호출된 엔드포인트에 따라 API 서버의 `healthy`/`live`/`ready` 상태를 나타낸다.
+아래 표시된 더 자세한 옵션은 운영자가 클러스터나 특정 API 서버의 상태를 디버그하는데 사용할 수 있다.
+
+아래 예시는 헬스 API 엔드포인트와 상호 작용할 수 있는 방법을 보여준다.
+
+모든 엔드포인트는 `verbose` 파라미터를 사용하여 검사 항목과 상태를 출력할 수 있다.
+이는 운영자가 머신 사용을 위한 것이 아닌, API 서버의 현재 상태를 디버그 하는데 유용하다:
```

Comment:
```suggestion
이는 머신이 사용하기 위한 것이 아니라, 운영자가 API 서버의 현재 상태를 디버깅하는데 유용하다.
```

- 문장부호 콜론(:)의 사용법이 영어와 국어에 차이가 있어서 대체로 국문 번역 시 생략하는 것이 자연스럽다고 합니다. 현재까지 번역된 문서의 경우 :을 모두 생략하고 있으므로 생략이 필요합니다. 생략 후에는 문장의 끝을 나타내는 .로 변경하시면 됩니다.
- 원문을 참고하여 좀 더 자연스럽게 수정해봤습니다.
>This can be useful for a human operator to debug the current status of the Api server, it is not intended to be consumed by a machine: